thirdparty/vars.sh

```diff
@@ -372,7 +378,8 @@ SIMDJSON_SOURCE=simdjson-1.0.2
 SIMDJSON_MD5SUM="5bb34cca7087a99c450dbdfe406bdc7d"
 
 # all thirdparties which need to be downloaded is set in array TP_ARCHIVES
-export TP_ARCHIVES="LIBEVENT
+export TP_ARCHIVES="LIBBACKTRACE
```

Comment:
if you add a lib, and use it in CMake by default , you should check it at build.sh